tensorly/decomposition/_parafac2.py

```diff
@@ -115,6 +117,9 @@ def _parafac2_reconstruction_error(tensor_slices, decomposition, norm_matrices=N
         * projections : List of projection matrices used to create evolving factors.
     norm_matrices : float, optional
         The norm of the data. This can be optionally provided to avoid recalculating it.
+    projected_tensor : ndarray, optional
+        The projections of X into an aligned tensor for CP decomposition. This can be optionally
```

Comment:
If you can, rebasing onto TensorLy master will get rid of these changes that aren't associated with this PR.

Reply:
Something weird happened when we rebased before submitting the PR so we got many other commits included as well (which we fixed with a new rebase just now). However we did make the changes in b12844ff49684741899247f5df7f81d44f9f864b (commit 1) here. They were small enough that we didn't bother to create a separate PR for them.

What do you think? Is it fine to include the first commit in this PR?